src/Libraries/PythonNodeModels/Docs/PythonMigrationWarning.html

```diff
@@ -100,6 +100,18 @@
         display: none;
     }
 
+    .button {
+        background-color: transparent;
+        border-color: #3c3c3c;
+        color: #bbbbbb;
+    }
+
+    .button:hover {
+        border-color: #656565;
+        background-color: #373737;
+        opacity: 1.0;
+    }
+
     @media screen and (max-width: 600px) {
```

Comment:
Makes sense but I wonder why we are using media queries when the panel was not even extendable until recently. Did we copy this content from somewhere else?

Reply:
@mmisol These are all produced by MatterLab. Not so sure

Reply:
@mmisol Do you see any issue around the styling here around extend behavior? I can log follow up issue if you found any bug

Reply:
Sorry, I did not understand the question. I do think the media query should be removed unless it actually provides noticeable improvements. At least to avoid the pitfall of running into similar issues again in the future.

Reply:
I agree. Looking into it. I was just wondering if you already noticed any bug

Reply:
This is what I saw as difference
![image](https://user-images.githubusercontent.com/3942418/99818169-e15cd180-2b1b-11eb-9281-874fdfd64817.png)

Reply:
I think the style difference in this case is nice to have so I'm ok with keeping the media query. Thanks for checking @QilongTang !

Reply:
@mmisol Thanks for the checking, I did not notice this before